picasso/src/main/java/com/squareup/picasso/Stats.java

```diff
@@ -75,8 +77,13 @@ void dispatchCacheMiss() {
     handler.sendEmptyMessage(CACHE_MISS);
   }
 
+  @TargetApi(Build.VERSION_CODES.JELLY_BEAN_MR2)
```

Comment:
This isn't how this annotation is meant to be used. You should call a method inside the conditional for the API-specific behavior and then have this annotation on it. With how you have it now you could use API 18+ methods in the `else` block and lint wouldn't complain.